OpenRA.Mods.Common/AI/AIHarvesterModule.cs

```diff
@@ -20,54 +20,52 @@
 
 namespace OpenRA.Mods.Common.AI
 {
-	class AIHarvesterManager
+	[Desc("Manages AI harvesters to ensure they always continue harvesting as long as there's resources on the map.")]
+	public class AIHarvesterModuleInfo : IAIModuleInfo
 	{
-		readonly HackyAI ai;
+		[Desc("Name for identification purposes.")]
+		public readonly string Name = "default-harvester-module";
```

Comment:
IIRC the other PR doesn't have `default-` in the harvester module's name, which do we want?

Reply:
The plumbing has out-evolved this Name property in the split off PRs